frontend/ui/widget/configdialog.lua

```diff
@@ -994,22 +996,23 @@ function ConfigDialog:onConfigFineTuneChoose(values, name, event, args, events,
         end
         if values then
             local value
+            local step = params.value_step or 1
```

Comment:
I get a crash if i try to press a `+` or `-`  in the bottom menu
`./luajit: frontend/ui/widget/configdialog.lua:999: attempt to index upvalue 'params' (a nil value)`
fixed by:
```suggestion
            local step = params and params.value_step or 1
```